releasenotes/notes/2.0/remove-result-mitigators-e9b97fb2f9349b1c.yaml

```
@@ -4,6 +4,5 @@ upgrade_misc:
     The ``qiskit.result.mitigation`` module has been removed following its deprecation in Qiskit 1.3.
     The removal includes the ``LocalReadoutMitigator`` and ``CorrelatedReadoutMitigator`` classes 
     as well as the associated utils. 
-
-    There is no alternative path in Qiskit, as their functionality had been superseded by the mthree package, 
+    There is no alternative path in Qiskit, as their functionality had been superseded by the ``mthree`` addon, 
```

```suggestion
    There is no alternative path in Qiskit, as their functionality had been superseded by the ```mthree`` addon. <https://github.com/Qiskit/qiskit-addon-mthree>`__
```